qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -170,10 +254,45 @@ def run(
             api_job_share_level = ApiJobShareLevel.NONE
 
         validate_job_tags(job_tags, IBMQBackendValueError)
+
+        if isinstance(circuits, (QasmQobj, PulseQobj)):
+            warnings.warn("Passing a Qobj to Backend.run is deprecated and will "
+                          "be removed in a future release. Please pass in circuits "
+                          "or pulse schedules instead. The `qobj` parameter has been "
+                          "renamed to `circuits`.", DeprecationWarning,
```

Comment:
You don't need the last sentence here, the deprecate_arguments decorator will raise a separate warning if `qobj` is used as a named parameter.